Vostok.Metrics.AspNetCore/Tls/TlsConnectionMetrics.cs

```diff
@@ -0,0 +1,41 @@
+﻿using JetBrains.Annotations;
+
+namespace Vostok.Metrics.AspNetCore.Tls;
+
+/// <summary>
+/// <para>Set of counters associated with the Tls.</para>
+/// </summary>
+[PublicAPI]
+public class TlsConnectionMetrics
```

Comment:
Shouldn't we also add the `CurrentTlsSession` property (the sum of existing versions)?

Reply:
I think, we can get this value from `all-tls-sessions-open` counter.